docs/data/joy/components/typography/typography.md

```diff
@@ -204,15 +201,25 @@ declare module '@mui/joy/styles' {
 
 ## Accessibility
 
-Here are a few tips to make sure you have an accessible typography component:
+Here are some factors to ensure that your Typography components are accessible:
 
-- **Color contrast**: always provide enough contrast between text and background. The minimum recommended [WCAG 2.0 color contrast ratio](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) is 4.5:1.
-- **Dynamic units**: use [relative units (rem)](/material-ui/customization/typography/#font-size) for `fontSize` to accommodate the user's settings.
-- **Heading hierarchy**: [don't skip](https://www.w3.org/WAI/tutorials/page-structure/headings/) heading levels.
-  Remember to [separate the semantics from the style](#changing-the-semantic-element).
+- Ensure sufficient color contrast between text and background, using a minimum of [WCAG 2.0's color contrast ratio](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) of 4.5:1.
+- Use [relative units](/material-ui/customization/typography/#font-size) such as rem for `fontSize` to accommodate the user's settings.
+- Use a consistent [heading hierarchy](https://www.w3.org/WAI/tutorials/page-structure/headings/), and avoid skipping levels.
+- Keep semantics and style separate by using the appropriate semantic elements(#semantic-elements).
 
 ## Common examples
 
 Examples showcasing how to compose designs with the `Typography` component and others as decorators.
 
 {{"demo": "DecoratorExamples.js"}}
+
+## Anatomy
```

Comment:
In this section, it's helpful to explain what the component looks like in its default state. How does it render in the DOM when I just drop a `<Typography>Hello world</Typography>` in my app with no props?

The Typography component is unique since its default state renders to a `<p>`, but if you nest one inside of another, the nested one will render as a `<span>` so that it stays semantically correct. I think this is worth showing and explaining here, maybe with a link pointing to the "Nested Typography" section for complete details.